core/controllers/handler_schema_constants.py

```diff
@@ -0,0 +1,157 @@
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Handlers which require schema validation, but currently they do
+# not have schema. In order to add schema incrementally this list is
+# maintained. Please remove the name of the handlers if they already
+# contains schema.
+
+"""Contains a list of handler class names that do not contain schema."""
+
+from __future__ import annotations
+from __future__ import unicode_literals
+
+HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS = [
+    'AnswerSubmittedEventHandler',
+    'AssetDevHandler',
+    'AudioUploadHandler',
+    'BulkEmailWebhookEndpoint',
+    'DeferredTasksHandler',
+    'DeleteAccountPage',
+    'EditableQuestionDataHandler',
+    'EditableSkillDataHandler',
+    'EditableStoryDataHandler',
+    'EditableSubtopicPageDataHandler',
+    'EditableTopicDataHandler',
+    'ExplorationActualStartEventHandler',
+    'ExplorationCompleteEventHandler',
+    'ExplorationEmbedPage',
+    'ExplorationMaybeLeaveHandler',
+    'ExplorationStartEventHandler',
+    'ExportAccountHandler',
+    'FeedbackThreadStatusChangeEmailHandler',
+    'FetchSkillsHandler',
+    'FlagExplorationEmailHandler',
+    'FlagExplorationHandler',
+    'IncomingReplyEmailHandler',
+    'InstantFeedbackMessageEmailHandler',
+    'JobOutputHandler',
+    'JobsHandler',
+    'LearnerAnswerDetailsSubmissionHandler',
+    'LearnerGoalsHandler',
+    'LearnerIncompleteActivityHandler',
+    'LeaveForRefresherExpEventHandler',
+    'MemoryCacheAdminHandler',
+    'MemoryCacheHandler',
+    'MergeSkillHandler',
+    'NewSkillHandler',
+    'NewTopicHandler',
+    'NotificationHandler',
+    'NotificationsDashboardHandler',
+    'NotificationsDashboardPage',
+    'NotificationsHandler',
+    'OldNotificationsDashboardRedirectPage',
+    'PendingAccountDeletionPage',
+    'PreferenceHandler',
+    'PreferencesHandler',
+    'PretestHandler',
+    'ProfileHandler',
+    'ProfilePage',
+    'PromoBarHandler',
+    'QuebstionsListHandler',
+    'QuestionCountDataHandler',
+    'QuestionCreationHandler',
+    'QuestionPlayerHandler',
+    'QuestionSkillLinkHandler',
+    'QuestionsListHandler',
+    'RatingHandler',
+    'ReaderFeedbackHandler',
+    'RecentCommitsHandler',
+    'RecommendationsHandler',
+    'ReleaseCoordinatorPage',
+    'ResubmitSuggestionHandler',
+    'ReviewableSuggestionsHandler',
+    'SignupHandler',
+    'SignupPage',
+    'SiteLanguageHandler',
+    'SkillDataHandler',
+    'SkillDescriptionHandler',
+    'SkillEditorPage',
+    'SkillMasteryDataHandler',
+    'SkillRightsHandler',
+    'SkillsDashboardPageDataHandler',
+    'SolutionHitEventHandler',
+    'StartedTranslationTutorialEventHandler',
+    'StateCompleteEventHandler',
+    'StateHitEventHandler',
+    'StatsEventsHandler',
```

Comment:
This already has schema, you need to always update this with the current data otherwise the tests will fail.

Reply:
done